tests/collect_test.rs

```diff
@@ -76,8 +76,8 @@ fn collect_test_eval_groups() {
         println!("fast: {:?} = {:?}", s, res.value);
 
         match &s[..] {
-            "abdef" => assert_eq!(res.value, vec![4u128, 4u128, 4u128]),
-            "gZ???" => assert_eq!(res.value, vec![3u128, 3u128, 3u128]),
```

Comment:
@jimouris Please confirm that we expect the results to change as I've done here.

Also, what are "abdef" and "gZ???"?

Reply:
Yup, looks good. We always append the count at the end.

> Also, what are "abdef" and "gZ???"?
two random strings. (inherited from Poplar)